wordpress-mobile/WordPress-iOS
# Data: Organizes core data entity graph.

As part of an exercise to review entity relationships I organized the our model's entity graph.  This PR contains the changes.  I figured we might save them for posterity. Maybe? Maybe not? 

To test:
Build the app. Confirm no unexpected issues or warnings when building and the app builds successfully. 
Select the data model and switch to the Style tab from the editor.  Confirm the tables are laid out in a readable graph rather than stacked one atop another.

Example:
<img width="1728" alt="Screen Shot 2020-05-21 at 1 42 53 PM" src="https://user-images.githubusercontent.com/1435271/82594257-6b29e300-9b69-11ea-9041-990947984921.png">


@leandroalonso would you be game to review this one?

PR submission checklist:
